### PR TITLE
[Security] Add missing German translations

### LINKED ISSUE #41045: Missing translations for German (de)
Hello,

There are some missing translation for German (de). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #41045
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf [hunk 1]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Ungültiger oder abgelaufener Anmelde-Link.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Zu viele fehlgeschlagene Anmeldeversuche, bitte versuchen Sie es in einer Minute noch einmal.</target>
+            </trans-unit>
+            <trans-unit id="20">
+                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
+                <target>Zu viele fehlgeschlagene Anmeldeversuche, bitte versuchen Sie es in in %minutes% Minuten noch einmal.</target>
```
Comment (jschaedl) [inline src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf]:
```suggestion
                <target>Zu viele fehlgeschlagene Anmeldeversuche, bitte versuchen Sie es in %minutes% Minuten noch einmal.</target>
```

Comment (derrabus) [reply src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf]: 🙈 Thanks

**Review by jschaedl — APPROVED** [review]

**Comment by Nyholm** [comment]
@xabbuh Please review

**Review by malteschlueter — APPROVED** [review]

File: src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf [hunk 2]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Ungültiger oder abgelaufener Anmelde-Link.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Zu viele fehlgeschlagene Anmeldeversuche, bitte versuchen Sie es in einer Minute noch einmal.</target>
```
Comment (xabbuh) [inline src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf:75]: Why do we have a `%minutes%` placeholder in the message but do not reuse it in the translation?

Comment (malteschlueter) [reply src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf:75]: Doesn't it sound nicer like that in German for a singular sentence?

Comment (derrabus) [reply src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf:75]: Yes, that's what I thought as well. The value should be `1` at this point.

Comment (xabbuh) [reply src/Symfony/Component/Security/Core/Resources/translations/security.de.xlf:75]: Fine for me if we can be sure that the values is always 1.

**Review by xabbuh — APPROVED** [review]

**Review by dmaicher — APPROVED** [review]

**Pull request merged**